src/table/src/TableRow.js

```diff
@@ -144,8 +144,8 @@ class TableRow extends PureComponent {
           innerRef={this.onRef}
           className={cx(themedClassName, className)}
           display="flex"
-          aria-selected={isHighlighted}
-          aria-current={isSelected}
+          data-ishighlighted={isHighlighted}
+          aria-selected={isSelected}
```

Comment:
Downshift adds `aria-selected` and it behaves like `isSelected`. Using both `aria-current` and `aria-selected` also probably isn't correct accessibility wise so I switched `isHighlighted` to a data attribute. I tested in a screen reader and highlights are still being announced correctly by Downshift.

Reply:
Yeah those `aria-` attributes keep tripping me up. We might need to check if this works for all the consumers of `TableRow`.

Reply:
I just tested `Menu`, which from what I can tell is the only other consumer of `createRowAppearance` (nothing else uses `TableRow`), and the highlighted / selected states are still working there.

Reply:
`Table` itself also seems to be working fine.